.eslintrc.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
   env: {
     browser: true
   },
+  globals: {
```

Comment:
would we add this in `test/.eslintrc.js` instead so that we only define the global in the test environment?